pkg/database/migrations/10_image_tables.up.sql

```diff
@@ -0,0 +1,514 @@
+    `synopsis`,
+    `checksum`,
+    `url`,
+    `created_at`,
+    `updated_at`
+  FROM `_movies_old`;
+
+INSERT INTO `scraped_items`
+  (
+    `id`,
+    `title`,
+    `description`,
+    `url`,
+    `date`,
+    `rating`,
+    `tags`,
+    `models`,
+    `episode`,
+    `gallery_filename`,
+    `gallery_url`,
+    `video_filename`,
+    `video_url`,
+    `studio_id`,
+    `created_at`,
+    `updated_at`
+  )
+  SELECT
+    `id`,
+    `title`,
+    `description`,
+    `url`,
+    `date`,
+    `rating`,
+    `tags`,
+    `models`,
+    `episode`,
+    `gallery_filename`,
+    `gallery_url`,
+    `video_filename`,
+    `video_url`,
+    `studio_id`,
+    `created_at`,
+    `updated_at`
+  FROM `_scraped_items_old`;
+
+-- these tables are a direct copy
+INSERT INTO `galleries` SELECT * from `_galleries_old`;
+INSERT INTO `performers_scenes` SELECT * from `_performers_scenes_old`;
+INSERT INTO `scene_markers` SELECT * from `_scene_markers_old`;
+INSERT INTO `scene_markers_tags` SELECT * from `_scene_markers_tags_old`;
+INSERT INTO `scenes_tags` SELECT * from `_scenes_tags_old`;
+INSERT INTO `movies_scenes` SELECT * from `_movies_scenes_old`;
+
+-- populate covers in separate table
+CREATE TABLE `scenes_cover` (
+  `scene_id` integer,
+  `cover` blob not null,
+  foreign key(`scene_id`) references `scenes`(`id`) on delete CASCADE
+);
+
+CREATE UNIQUE INDEX `index_scene_covers_on_scene_id` on `scenes_cover` (`scene_id`);
+
+INSERT INTO `scenes_cover` 
+  (
+    `scene_id`,
+    `cover`
+  )
+  SELECT `id`, `cover` from `_scenes_old` where `cover` is not null;
+
+-- put performer images in separate table
+CREATE TABLE `performers_image` (
+  `performer_id` integer,
+  `image` blob not null,
+  foreign key(`performer_id`) references `performers`(`id`) on delete CASCADE
+);
+
+CREATE UNIQUE INDEX `index_performer_image_on_performer_id` on `performers_image` (`performer_id`);
+
+INSERT INTO `performers_image` 
+  (
+    `performer_id`,
+    `image`
+  )
+  SELECT `id`, `image` from `_performers_old` where `image` is not null;
+
+-- put studio images in separate table
+CREATE TABLE `studios_image` (
+  `studio_id` integer,
+  `image` blob not null,
+  foreign key(`studio_id`) references `studios`(`id`) on delete CASCADE
+);
+
+CREATE UNIQUE INDEX `index_studio_image_on_studio_id` on `studios_image` (`studio_id`);
+
+INSERT INTO `studios_image` 
+  (
+    `studio_id`,
+    `image`
+  )
+  SELECT `id`, `image` from `_studios_old` where `image` is not null;
+
+-- put movie images in separate table
+CREATE TABLE `movies_images` (
+  `movie_id` integer,
+  `front_image` blob not null,
+  `back_image` blob,
+  foreign key(`movie_id`) references `movies`(`id`) on delete CASCADE
+);
+
+CREATE UNIQUE INDEX `index_movie_images_on_movie_id` on `movies_images` (`movie_id`);
+
+INSERT INTO `movies_images` 
+  (
+    `movie_id`,
+    `front_image`,
+    'back_image'
+  )
+  SELECT `id`, `front_image`, 'back_image' from `_movies_old` where `front_image` is not null;
```

Comment:
`'back_image'` here adds string back_image instead of blob to db
should be backtick like the rest 
it took me  a while to figure out why the back_images where missing after the migration :-)